Tools/clinic/clinic.py

```diff
@@ -4350,7 +4350,7 @@ def parse_arg(self, argname: str, displayname: str, *, limited_capi: bool) -> st
                     {bad_argument}
                     goto exit;
                 }}}}
-                {paramname} = PyUnicode_AsUTF8({argname});
+                {paramname} = PyUnicode_AsUTF8Safe({argname});
```

Comment:
There should be different code for `limited_capi` is true.

Reply:
Oh. I wanted to add PyUnicode_AsUTF8Safe() to the limited C API in a separated PR. Maybe it's more convenient to do it in the PR, since there is a lot of code generated by Argument Clinic impacted by these changes.